src/resources/marketplace.js

```diff
@@ -15,18 +15,23 @@ const storeKeys = {
 }
 
 class Marketplace extends Adaptable {
+
   constructor({
     contractService,
     ipfsService,
-    fetch,
     indexingServerUrl,
```

Comment:
Let's get rid of indexingServer in the origin-js codebase - it is not used by anything AFAICT.